tools/cpplint.py

```diff
@@ -46,7 +45,7 @@
 import getopt
```

Comment:
This file probably should be excluded from formatting, it is an external tool just included here for convenience I presume.